### PR TITLE
add first homework

### PR DESCRIPTION
**Pull request by NOcode-developer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,3 +64,6 @@ typings/
 
 karma.conf.js
 protractor.conf.js
+
```
Comment (vladyslav-yermolin-moc) [inline .gitignore:67]: Hey, @NOcode-developer, please revert the changes of this file, to remove it from this pull-request.
 In this task, you should only add your Readme, not change the settings of whole repo.

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (NOcode-developer) [reply .gitignore]: I made changes to the .gitignore file.
Please check if everything is done correctly.

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Comment by vladyslav-yermolin-moc** [comment]
Well done!
Please resubmit your task in Google classroom
